Pages/CourseList/script.js

```diff
@@ -0,0 +1,89 @@
+import { baseUrl } from "../../config.js";
+const accessToken = localStorage.getItem("token");
+if (!accessToken) window.location.href = "../LoginPage/Index.html";
+
+document.addEventListener("DOMContentLoaded", () => {
+    // Define the API endpoint
+
+    // Func. de dar fetch na API
+    function fetchCourses() {
+        fetch(`${baseUrl}/cursos/cursos`, {
+            method: "GET",
+            headers: {
+                Accept: '"application/json"',
+                Authorization: `Bearer ${accessToken}`,
+            },
+        })
+            .then((response) => {
+                if (response.ok) {
+                    console.log("Response:  " + response);
+                    return response.json();
+                } else {
+                    throw new Error("Network response was not ok");
+                }
+            })
+            .then((data) => {
+                console.log("Data: ", data.cursos);
+                displayCourses(data.cursos);
+            })
+            .catch((error) => {
+                console.error(
+                    "There was a problem with the fetch operation:",
+                    error
+                );
+            });
+    }
+
+    // Manipulação DOM
+    function displayCourses(courses) {
+        const videosContainer = document.querySelector(".videos-container");
+
+        // Tirar o que tem no HTML
+        videosContainer.innerHTML = "";
+
+        courses.forEach((course) => {
```

Comment:
```suggestion
        courses.forEach((course) => {
```
como vai fazer um for each e construir os items do curso por vez, poderia ter uma função pra algo do tipo
buildCourseItem que recebe o curso e monta